### PR TITLE
Reintegrate communities

### PR DESCRIPTION
**Pull request by DanielKambich**
I think I was able to cover a majority of integration and did some minor UI tweaks. Let me know if I missed anything

### CODE REVIEW

File: src/components/communities/CommunityAdministrationPage.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from "react";
-import { ScrollView, Switch, StyleSheet, Alert, Button } from "react-native";
+import { ScrollView, Switch, StyleSheet, Alert } from "react-native";
 import { SubmitHandler, useForm } from "react-hook-form";
+import { useTheme } from "@shopify/restyle";
```
Comment (stephend017) [inline src/components/communities/CommunityAdministrationPage.tsx:4]: do we need this line? I think you import `theme` below and I haven't seen us use this hook in the codebase yet

Comment (DanielKambich) [reply src/components/communities/CommunityAdministrationPage.tsx:4]: This is used in all the themed components. The import of Theme below is the type of theme we define in theme.tsx. I am using it here to give our 'red' buttons consistency.

**Review by stephend017 — APPROVED** [review]
LGTM!

File: src/components/communities/CommunityAdministrationPage.tsx (hunk 1 above)
Comment (stephend017) [reply src/components/communities/CommunityAdministrationPage.tsx:4]: ah I see. I've been just importing the `theme` object directly from theme.tsx but this is probably better

**Pull request merged**